### PR TITLE
Only add modified `PATH` to login files in login shells

### PR DESCRIPTION
**Pull request by chrysle**
Part of #46

### CODE REVIEW

**Comment by ofek** [comment]
Should we modify only the RC files instead as I think that is what other programs do? Just let me know your reasoning because I don't have time to reread that other thread please

**Comment by chrysle** [comment]
From https://github.com/ofek/userpath/issues/3#issuecomment-661917621:

> I believe the best way to treat bash users is probably to assume that the user will encounter at least one "login shell" somewhere during the login process before running userpath, and add our PATH definition to the first dotfile for bash login shells that we find. The reason I say this is that these files are also where Red Hat, Ubuntu and others suggest that users should add their own $PATH definitions.

**Comment by chrysle** [comment]
@ofek Could you take another look?

**Review by ofek — APPROVED** [review]

File: userpath/shells.py [hunk 1]
```diff
@@ -44,13 +42,12 @@ def config(self, location, front=True):
             # Terminal.app runs a login shell by default for each new terminal window.
             login_config = bash_profile_path
 
-        configs[login_config] = contents
-
-        return configs
+        # return {login_config: contents, path.join(self.home, '.bashrc'): contents}
```
Comment (ofek) [inline userpath/shells.py:45]:
```suggestion
```

**Comment by ofek** [comment]
Is this sufficient to close the associated issues now?

**Comment by chrysle** [comment]
Thanks! I aim to still introduce the command line option and environment variable.

**Comment by ofek** [comment]
Oh great thank you!

**Pull request merged**